benchmarks/asv/app_suite.py

```diff
@@ -31,3 +37,202 @@ def time_hydrogen(self, n_steps, optimize):
         hyperparams = {'n_steps': n_steps,
                        'optimize': optimize}
         benchmark_vqe(hyperparams)
+
+    def peakmem_hydrogen(self, n_steps, optimize):
+        """Benchmark the peak memory usage of the VQE algorithm with the UCCSD ansatz for computing
+         the ground state energy of the hydrogen molecule."""
+        hyperparams = {'n_steps': n_steps,
+                       'optimize': optimize}
+        benchmark_vqe(hyperparams)
+
+
+class VQE_heavy:
+    """Benchmark the VQE algorithm using different grouping options for the lithium hydride molecule
+     with 2 active electrons and 8 active spin-orbitals. The sto-3g basis set and UCCSD ansatz are
+     used."""
+
+    params = ([3], [False, True])
+    param_names = ['n_steps', 'optimize']
+
+    H_coeffs = np.array([-6.74845266e+00, -1.02553930e-01, 1.00530907e-02, 1.00530907e-02,
```

Comment:
Could we move these to a separate file that we read in?

Reply:
Yes, definitely.

Reply:
How about putting these data in a `data.py` file and then use something like
`from data.py import H_coeffs`